### PR TITLE
Feature/account settings form

### PR DESCRIPTION
**Pull request by Patrick-G-Oliver**
# Pull Request Template

## Description
Add a summary that address the following:
motivation/purpose for the feature
Modifications made to reflect changes in the Figma document (i.e. stakeholders' request for design changes)
What were those changes
Account settings change-password and change-email inputs and buttons modified with modals added

## Commits Checklist
- [ ] commits have clear title names 
- [ ] commit has descriptive message of what has changed 
- [ ] more small commits than less big commits 

## Code Checklist 
- [ ] code is formatted appropriately 
- [ ] code meets DRY standards
- [ ] no dead code (check for logs and print statements)  
- [ ] no TODOS instead necessary comments 
- [ ] code follows Story Squad standards for:
     - Language 
     - Framework 
     - Libraries   

### PRs needs to be reviewed by at least 2 team members
 - TPL is the 3rd reviewer 
 - Lots of comments and suggestions, please! 

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by leachtucker — APPROVED** [review]
Nice work, Patrick! The modals look great

**Review by MosharrafMusa — APPROVED** [review]
Looks like you fixed the issues

File: src/components/pages/AccountSettingsForm/ChangeEmailModal.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    props.setEmailModalVisible(true);
+  };
+
+  const handleCancel = () => {
+    props.setEmailModalVisible(false);
+  };
+
+  return (
+    <Modal
+      visible={props.emailModalVisible}
```
Comment (Robert-Lark) [inline src/components/pages/AccountSettingsForm/ChangeEmailModal.js:17]: nice job figuring out these modals - that's a steep learning curve.

File: src/styles/less/AccountSettingsForm.less [hunk 2]
```diff
@@ -97,4 +97,117 @@
...
+    }
+}
+
+.modal-header-text {
+    font-size: 1.25rem;
+    font-weight: 700;
+    margin: 2vh 0 3vh 0;
+    @media screen and (min-width: 750px) {
+        margin-bottom: 2vh;
+    }
```
Comment (Robert-Lark) [inline src/styles/less/AccountSettingsForm.less:131]: good work on the media design.

**Review by Robert-Lark — APPROVED** [review]
Great work

**Pull request merged**